step-certificates/templates/configmaps.yaml

```diff
@@ -107,7 +107,7 @@ data:
       --provisioner "{{.Values.ca.provisioner.name}}" \
       --with-ca-url "{{include "step-certificates.url" .}}" \
       --password-file "$TMP_CA_PASSWORD" \
-      --provisioner-password-file "$TMP_CA_PROVISIONER_PASSWORD" {{ if not .Values.ca.db.enabled }}--no-db{{ end }}
+      --provisioner-password-file "$TMP_CA_PROVISIONER_PASSWORD" {{ if not .Values.ca.db.enabled }}--no-db{{ end }} {{ if .Values.ca.bootstrap.rootCA.secret }}--root /tmp/certs/tls.crt --key /tmp/certs/tls.key{{ end }}
```

Comment:
Can you add it in a new line, and now that you're on it, can you also add the `--no-db` to a new line :)

Reply:
Actually, not sure how as `\` has to be on previous line, but only if values are available. Not sure how to parametrize new line properly. Any idea ?

Reply:
As we only adding one volume, we can not just use bash for this without making it complex, so I think you should name it in the way that `step` names those files when you do `step ca init`, so `root_ca.crt` and `root_ca_key`.

You can add variables for those, and you can leave those names as default because you can do the `if` with the `rootRef`.